app/views/layouts/_guide.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render partial: 'layouts/runner_assets', locals: {language: @guide.language} %>
+<%= render partial: 'layouts/runner_assets', locals: {language: @guide.language, prefix: 'assets'} %>
```

Comment:
Please extract this to a view helper in order to reduce duplicated logic. Something like 

```erb
<%= render_runner_assets @guide.language, :layout %> 
<%= render_runner_assets @exercise.language, :editor %>
<%= render_runner_assets @exercise.language, :submit %>
```